vulcan/lib/client/jsx/components/workflow/user_interactions/inputs/input_types.ts

```diff
@@ -15,6 +15,30 @@ export type InputType = string;
 // This type represents the unflattened raw data format coming in from the cwl.
 export type DataEnvelope<Inner> = { [k: string]: Inner };
 
+export function get_vals(de: DataEnvelope<any>): any[] {
```

Comment:
There's a function that already does this, `Object.values(de)`.  I'd use that and replace this function.

Reply:
Tried this, but it wasn't giving the right format?  Maybe inexperience and just how the developer tools console renders, but I thought It's output was still keyed, rather than an Array...

Will do / try putting this and the others below back as requested.